components/homme/src/share/kinds.F90

```diff
@@ -29,12 +29,19 @@ module kinds
   long_kind    = 8,                      &
   log_kind     = 4,                      &
   real_kind    = 8,                      &
-  iulog        = 6,                      & ! stderr file handle
 #if HOMME_QUAD_PREC
   longdouble_kind    = 16
 #else 
   longdouble_kind    = 8
 #endif
+
+#ifdef SCREAM
```

Comment:
Here, you're in the case of !CAM and SCREAM. Does this include a CIME-based build or just a standalone SCREAM build? If it includes a CIME-based build, shouldn't we using the CAM block above?

Reply:
Well, this should include _any_ SCREAM build, cime or not cime.

I tried to execute the CAM block (without iulog, since SCREAM deos not build cam_logfile...although it could, since it's basically nothing), but then I had issues b/c QUAD_PREC was enabled and I wasn't getting the longdouble kind. I could disable QUAD_PREC when building homme within scream, I guess, and then we could just use the 1st block.

Reply:
Ok. No changes needed. I'm a little concerned that we're not using shr_kind_mod values when building under CIME, but that's orthogonal to this PR.

Reply:
Yeah, that's a good point. This is how it was before, so I figured I didn't touch that, but it is a bit concerning. I can add an extra commit addressing that.

Reply:
Ok, I changed it. Gotta retest now.